goalgorilla/open_social
# Issue #3366365 by nkoporec: Tag text is cut when using social_follow_tag module and layout builder

## Problem
When using the social_follow_tag module together with layout builder and with 2 column layout then the tag text is cut and not displayed correctly.

## Solution
 Adjust the CSS so that the tag text is not cut off.

## Issue tracker
https://www.drupal.org/project/social/issues/3366365

## Theme issue tracker
*[Required if applicable] Paste a link to the drupal.org theme issue queue item, either from [socialbase](https://www.drupal.org/project/socialbase) or [socialblue](https://www.drupal.org/project/socialblue). If any other issue trackers were used, include links to those too.*

## How to test
1. Enable social_follow_tag module and create a really long tag name.
2. Enable layout builder
3. Create a new page with layout builder and set a 2 column display
4. In one of the columns add the Social follow tag block and put something in the other (like content block)
5. Check the tag display.

## Definition of done
### Before merge

### After merge

## Screenshots
Before:
![2023-06-13_08-09](https://github.com/goalgorilla/open_social/assets/35064680/fb8ac0a7-2ad1-4772-ae9c-d7a1c11499b5)


After:
![2023-06-13_08-13](https://github.com/goalgorilla/open_social/assets/35064680/86e5a5b0-d7e8-4ebf-982f-94b0d94b1588)



## Release notes
Fixed a styling issue where tag text in social follow tag block would be cut off when using it with layout builder and 2 column layouts.

## Change Record


## Translations
